opencolorio_config_aces/config/generation/common.py

```diff
@@ -223,6 +223,64 @@ def view_transform_factory(name,
     return view_transform
 
 
+@required('OpenColorIO')
+def look_factory(name,
+                 process_space=None,
+                 description=None,
+                 forward_transform=None,
+                 inverse_transform=None,
+                 base_look=None):
+    """
+    *OpenColorIO* look factory.
+
+    Parameters
+    ----------
+    name : unicode
+        *OpenColorIO* look name.
+    process_space : unicode, optional
+        *OpenColorIO* look process space, e.g. *OpenColorIO* colorspace or role
+        name.
+    description : unicode, optional
+        *OpenColorIO* look description.
+    forward_transform : object, optional
+        *To Reference* *OpenColorIO* look transform.
+    inverse_transform : object, optional
+        *From Reference* *OpenColorIO* look transform.
+    base_look : ViewTransform, optional
+        Inherited *OpenColorIO* base look.
+
+    Returns
+    -------
+    Look
+        *OpenColorIO* look.
+    """
+
+    import PyOpenColorIO as ocio
+
+    if process_space is None:
+        process_space = ocio.ROLE_SCENE_LINEAR
```

Comment:
In the Gist, the process_space for BlueLightFix is being set to scene_linear, but that role is ACEScg in this config.  The process_space must be set to ACES2065-1 for this LMT.  And, in fact, anything that is called an LMT must use that process space, by definition.

Reply:
Ah, good catch!